PWGHF/HFC/Tasks/taskCharmHadronsFemtoDream.cxx

```diff
@@ -583,7 +583,7 @@ struct HfTaskCharmHadronsFemtoDream {
         part.tpcNSigmaPr(),
         part.tofNSigmaPr());
     }
-    if (!sliceCharmHad.size() || !sliceTrk1.size()) {
+    if (sliceCharmHad.size() || sliceTrk1.size()) {
```

Comment:
`size` does not return `bool`. Use explicit comparison with `0`.

Reply:
Thanks! fixed it!